template.yaml

```diff
@@ -77,11 +77,12 @@ Resources:
     Properties:
       FunctionName: !Sub 'CumulusAggFetchAuthorizer-${DeployStage}'
       Handler: src/handlers/site_upload/api_gateway_authorizer.lambda_handler
+      Layers: [arn:aws:lambda:us-east-1:336392948345:layer:AWSSDKPandas-Python311:17]
```

Comment:
ok, i don't think this :should: be happening, but as near as I can tell: the reason uploads (and also some other bits of the upload pipeline/dashboard apis) broke is because in python 3.11 lambdas, the isolation between lambdas is different? Layers referencing a handler which didn't use pandas were previously fine, but now it seems like, since every lambda can access the entire projects code, without this layer they fail on importing pandas?

I'll look into this and figure out if there's a better isolation pathway.

Reply:
But why would an `import pandas` line even be hit in this case? Is the lambda now running a different code path because of a layer change?

That seems concerning (but maybe could happen if we now are pulling in a newer version of some pip package that added a pandas dep?).

Did we accidentally introduce an unexpected dependency on pandas ourselves and that's what's causing this? Like, do we have stacktraces for where the import comes from?